<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>849</width>
    <height>651</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>50</y>
      <width>111</width>
      <height>17</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Detection Classifier:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>50</y>
      <width>131</width>
      <height>17</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Recognition Algorithm:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="inputPath">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>10</y>
      <width>701</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>./Images</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushOpenFolder">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>99</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>Open Folder</string>
    </property>
   </widget>
   <widget class="QComboBox" name="dropDownDetect">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>70</y>
      <width>261</width>
      <height>27</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>haarcascade_frontalface_alt</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>haarcascade_frontalface_default</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>haarcascade_frontalface_alt2</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>haarcascade_frontalface_alt_tree</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>haarcascade_profileface</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>lbpcascade_frontalface</string>
     </property>
    </item>
   </widget>
   <widget class="QComboBox" name="dropDownRecognize">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>70</y>
      <width>281</width>
      <height>27</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>LBP Histograms OpenCV</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Eigenfaces OpenCV</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Fisherfaces OpenCV</string>
     </property>
    </item>
   </widget>
   <widget class="QPushButton" name="pushDetect">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>110</y>
      <width>111</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>Detect Faces</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushRecognize">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>110</y>
      <width>131</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>Recognize Faces</string>
    </property>
   </widget>
   <widget class="QTextBrowser" name="outputText">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>140</y>
      <width>561</width>
      <height>401</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>580</x>
      <y>50</y>
      <width>81</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="frameShape">
     <enum>QFrame::NoFrame</enum>
    </property>
    <property name="text">
     <string>Benchmarking:</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushLoadAllRuns">
    <property name="geometry">
     <rect>
      <x>710</x>
      <y>70</y>
      <width>101</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>Load All Runs</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushLoadRun">
    <property name="geometry">
     <rect>
      <x>580</x>
      <y>70</y>
      <width>111</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>Load Single Run</string>
    </property>
   </widget>
   <widget class="QTextBrowser" name="outputRuns">
    <property name="geometry">
     <rect>
      <x>580</x>
      <y>140</y>
      <width>256</width>
      <height>401</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="pushCompareDetection">
    <property name="geometry">
     <rect>
      <x>580</x>
      <y>110</y>
      <width>111</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>Compare Detection</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushCompareRecognition">
    <property name="geometry">
     <rect>
      <x>710</x>
      <y>110</y>
      <width>121</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>Compare Recognition</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushSaveLog">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>550</y>
      <width>85</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>Save Log</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushClearA">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>550</y>
      <width>51</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>Clear</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushClearB">
    <property name="geometry">
     <rect>
      <x>780</x>
      <y>550</y>
      <width>51</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>Clear</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>849</width>
     <height>20</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFacetag_Framework">
    <property name="title">
     <string>Facetag Framework</string>
    </property>
   </widget>
   <addaction name="menuFacetag_Framework"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
